Libraries/Utilities/ImagePickerIOS.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
@JoeStanton ah, I wasn't sure about that. I had it on `.ios.js` before but felt that `IOS.ios.js` is a bit too much since the name implicates the OS already. 

`AlertIOS.js` for example is missing the `ios.js` which makes it even more confusing.